### DISCUSSION TITLE
ERROR: Could not build wheels for fairseq

### DISCUSSION THREAD

**Discussion by azeka152** [voicepaw/so-vits-svc-fork#277] [Q&A] [answered]
when I use the command "pip install -U so-vits-svc-fork" it ends with "ERROR: Could not build wheels for fairseq, which is required to install pyproject.toml-based projects". how to fix it? 
![image](https://user-images.githubusercontent.com/89282858/230816173-915e543d-83f3-4aa2-b2a4-14ee480f419a.png)

**Comment by 34j** [comment, marked as answer]
Duplicate of #142

**Comment by vinayicer** [comment]
nice
